controllers/openstackvmset_controller.go

```diff
@@ -852,6 +873,8 @@ func (r *OpenStackVMSetReconciler) vmCreateInstance(instance *ospdirectorv1beta1
 	fsMode := corev1.PersistentVolumeFilesystem
 	trueValue := true
 	terminationGracePeriodSeconds := int64(0)
+	// TODO: How do we account for custom roles here?
```

Comment:
do you mean custom tripleo roles? or VMs we just prepare but are not part of tripleo deployment?

Reply:
Yes, custom OOO roles.  In the playbook generator, we have the custom tarball and can now crack it open and examine `roles_data.yaml`, if any.  In that case of VMSets though, we don't have a window into these potential custom roles.  Maybe we should just always mount the minimal kubeconfig for Pacemaker and make sure its filesystem perms are as restrictive as possible (as you suggest in your other comment)?

Reply:
yes, as discussed on slack